tests/budget_accounting_test.py

```diff
@@ -95,13 +101,43 @@ class ComputeBudgetTestCase:
             mechanisms: []
 
         testcases = [
+            ComputeBudgetTestCase(
+                name="generic",
+                epsilon=0.22999925338484556,
+                delta=1e-5,
+                mechanisms=[
+                    ComputeBudgetMechanisms(
+                        1,
+                        6.41455078125,
+                        MechanismType.GENERIC,
+                        1,
+                        1,
+                        expected_mechanism_epsilon=0.2204717161227536,
```

Comment:
@dvadym 👋 This new type of test checks the mechanism epsilon and delta values.

Reply:
:thumbsup: